gen3-integration-tests/pyproject.toml

```diff
@@ -31,6 +32,9 @@ fastavro = "^1.8.2"
 psutil = "^5.9.8"
 nest-asyncio = "^1.6.0"
 allure-pytest = "^2.13.5"
+nextflowpy = "^0.11.0"
+boto3 = "<1.26.0"
+numpy = "<2.0.0"
```

Comment:
why do we need numpy?

Reply:
Documented the change in pyproject.toml ([here](https://github.com/uc-cdis/gen3-code-vigil/blob/4b7640dbaa5cc7d4717fef66d032011628fffa82/gen3-integration-tests/pyproject.toml#L39C1-L44C13)). Could you review and resolve this conversation if it looks good!

Reply:
I see that `test_argo_wrapper` imports pandas but the dependency is not even in pyproject.toml... Let's add it if you have a chance...